app/frontend/src/features/communities/CommunityPage/CommunityPage.tsx

```diff
@@ -161,6 +138,10 @@ export default function CommunityPage() {
     }
   }, [community, communitySlug, history]);
 
+  const { page = "overview" } = useParams<{ page: CommunityTab }>();
```

Comment:
```suggestion
  const { page = "overview" } = useParams<{ page: string }>();
```
Since this comes from the URL so it can be anything, hence the pattern below with `tab` to narrow it to the right typed string?